test/development/acceptance/ReactRefreshLogBox.test.ts

```diff
@@ -602,7 +602,7 @@ describe('ReactRefreshLogBox', () => {
          "label": "Build Error",
          "source": "./index.module.css
        Parsing css source code failed
-       Selector is not pure (pure selectors must contain at least one local class or id), (lightningcss, Selector(button, specificity = 0x1))",
+       Selector is not pure (pure selectors must contain at least one local class or id), (button)",
```

Comment:
It should look something like this:

```suggestion
       Selector "button" is not pure. Pure selectors must contain at least one local class or id.
```